skiko/src/macosMain/kotlin/org/jetbrains/skiko/Convertors.macos.kt

```diff
@@ -0,0 +1,54 @@
+package org.jetbrains.skiko
+
+import kotlinx.cinterop.useContents
+import platform.AppKit.*
+
+fun toSkikoEvent(
+    event: NSEvent,
+    kind: SkikoPointerEventKind,
+    view: NSView
+): SkikoPointerEvent {
+    var (xpos, ypos) = event.locationInWindow.useContents {
+        x to y
+    }
+    // Translate.
+    view.frame.useContents {
+       ypos = size.height - ypos
+    }
+    return SkikoPointerEvent(
+        xpos,
+        ypos,
+        toSkikoMouseButtons(event),
+        toSkikoModifiers(event),
+        kind,
+        event
+    )
+}
+
+fun toSkikoEvent(
+    event: NSEvent,
+    kind: SkikoKeyboardEventKind
+): SkikoKeyboardEvent {
+    return SkikoKeyboardEvent(
+        event.keyCode.toInt(),
+        toSkikoModifiers(event),
+        kind,
+        event
+    )
+}
+
+fun toSkikoMouseButtons(event: NSEvent): SkikoMouseButtons {
+    var result = 0
+    val mask = event.buttonMask.toInt()
+    if ((mask and 1) != 0 || event.buttonNumber == 0L) {
+        result = result.or(SkikoMouseButtons.LEFT.value)
+    }
+    if ((mask and 2) != 0 || event.buttonNumber == 1L) {
```

Comment:
Shalln't we process other buttons? I have a multi-button mouse and see the following NSEvent when pressing other buttons, if add smth like
```
override fun otherMouseDown(event: NSEvent) {
                skikoView?.onPointerEvent(toSkikoEvent(event, SkikoPointerEventKind.DOWN, nsView))
            }
            override fun otherMouseUp(event: NSEvent) {
                skikoView?.onPointerEvent(toSkikoEvent(event, SkikoPointerEventKind.UP, nsView))
            }
```

```
e=NSEvent: type=OtherMouseDown loc=(170,264) time=13656.9 flags=0 win=0x7feec27095f0 winNum=540 ctxt=0x0 evNum=3 click=1 buttonNumber=2 pressure=1 deviceID:0x0 subtype=0
PointerEvent: [EMPTY] + DOWN + [NONE]
e=NSEvent: type=OtherMouseUp loc=(170,264) time=13657.1 flags=0 win=0x7feec27095f0 winNum=540 ctxt=0x0 evNum=3 click=1 buttonNumber=2 pressure=0 deviceID:0x0 subtype=0
PointerEvent: [EMPTY] + UP + [NONE]
```
for pressing mouse wheel, and no events for other buttons on MX Master 3.